lib/ota/remote.txt

```diff
@@ -32,7 +32,7 @@ function table otaOnlineInit()
 {
     local Ota = table()
 
-    Ota["Online Updates Branch", string] = "Over-The-Air-Updates_Part-2-of-2_Online"
+    Ota["Online Updates Branch", string] = "Version-File-Scanning-Method-1"
```

Comment:
I need to revert this back to `Over-The-Air-Updates_Part2-of-2_Online` before merging.